index.js

```diff
@@ -153,5 +154,11 @@ module.exports = function (str, opts) {
 		str = str.replace(/^http:\/\//, '//');
 	}
 
+	var hasHttps = str.indexOf('https') === 0;
+
+	if (opts.normalizeSsl && hasHttps) {
+		str = str.replace(/^https/, 'http');
+	}
```

Comment:
Would be better to work on the parsed URL object further up in the code, so you could just replace `urlObj.protocol`.